tools/deploy_ipk.sh

```diff
@@ -1,23 +1,31 @@
 #!/bin/sh -e
 
-SSH_OPTIONS="-oBatchMode=yes"
+usage() {
```

Comment:
Regarding the commit message - I agree that in a product we should disable certification mode completely, and better to do that in compile time (we can simply make the certification getter/setter a NOP).
But for the time being, I think that RAX40 should have certification_mode enabled by default.

Reply:
We can definitely enable it when deploying on our targets, but enabling it by default from the prplWrt feed looks wrong to me.
In any case I think it doesn't hurt to have and use this option in CI, so if you agree let's keep it and we can discuss the default value in prplWrt later.

Reply:
Agree